client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -49,11 +49,41 @@
 
     private final int maxOutstandingCheckpointRequest;
 
+    @Getter
+    private final boolean isSubscriber;
```

Comment:
isSubscriber is perhaps a misleading flag for the user. instead we should say something to the effect of `subscribe for retention`.

Reply:
The name change has been made. Please have a look.